Microsoft.Toolkit.Uwp.SampleApp/SamplePages/CameraPreview/CameraPreviewCode.bind

```diff
@@ -0,0 +1,40 @@
+﻿<Page xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
This file should be split into xaml bind file so it can be edited by the user.

Reply:
Split into separate XAML bind file